content/en/docs/how-to/registry-authentication.md

```diff
@@ -7,74 +7,54 @@ weight: 1
+- Store credentials when you log in to a registry
+- Read credentials when accessing a registry to sign, verify, or display artifacts
+- Delete credentials when you log out of a registry
+ 
+You have three options for configuring a credentials store:
+1. Use the [`notation` config.json file]({{< ref "/docs/concepts/directory-structure" >}}).
+1. Use the Docker configuration file, which is located at `~/.docker/config.json`.
+1. Use the default credentials store for your operating system.
 
-`notation` uses [Docker Credential Helpers](https://github.com/docker/docker-credential-helpers) and its [protocol](https://docs.docker.com/engine/reference/commandline/login/#credential-helper-protocol) as the credential helpers. Currently, using Docker Credential Helpers requires manual installation and configuration in Linux by following the steps below. This configuration will be simplified in Notation v1.0.0.
+`notation` will resolve the credentials store in the order shown above. If no credential store is available and no other method is used for credentials, such as environment variables, `notation login` will not store the credentials and the command will fail.
 
-Install [Docker credential helper](https://github.com/docker/docker-credential-helpers) pass.
-
-```bash
-mkdir -p ~/bin
-curl -Lo ~/bin/docker-credential-pass "https://github.com/docker/docker-credential-helpers/releases/download/v0.7.0/docker-credential-pass-v0.7.0.linux-amd64"
-chmod +x ~/bin/docker-credential-pass
-```
-
-Generate and configure GPG key for encryption. 
+{{% alert title="Important" color="important" %}}
+If you have Docker installed on your system and used `docker login` to authenticate to your OCI-compliant registry, your credentials are already stored and available to `notation`. In this case, you don't need to run `notation login` again to authenticate to your OCI-compliant registry.
 
-{{% alert title="Note" color="primary" %}}
-If you have a GPP key already, run `gpg --edit-key` to trust your key instead.
+Docker also implements a fallback mechanism for storing your credentials. If no credentials store is available, Docker will store your credentials as base64 encoded values in the Docker configuration file. This is not secure and is not recommended for production environments. For more details, see [`docker login` default behavior](https://docs.docker.com/engine/reference/commandline/login/#default-behavior).
 {{% /alert %}}
 
-```console
-gpg --full-generate-key
-```
-
-Install and configure [pass](https://www.passwordstore.org/).
-
-```console
-sudo apt update 
-sudo apt install pass -y
-pass init $your_email
-```
-
-Configure Docker credential store in `~/.docker/config.json`.
+To configure a credentials store in the `notation` configuration file, add a `credStore` to the [`notation` config.json file]({{< ref "/docs/concepts/directory-structure" >}}). The following example shows `osxkeychain` as the configured credentials store.
```

Comment:
Should we move this paragraph above line 28? Because line 28~line 31 is talking about Docker, which is at the 2nd place in the order mentioned above.

Reply:
Thank you for the for the feedback.

I reread the section and i think it makes the most sense where it is.

This sentence is directly tied to the config sample below it and users should know about the `docker login` behavior before deciding how they want to setup their credentials.